src/wombats/game/processor.clj

```diff
@@ -70,23 +77,53 @@
         (add-occlusion-view game-state type)
         (add-custom-state game-state uuid type))))
 
+(defn- lambda-client
+  [{:keys [access-key-id secret-key]}]
+  (let [credentials (new BasicAWSCredentials access-key-id secret-key)]
+    (new AWSLambdaClient credentials)))
+
+(defn- lambda-request-body
+  [player-state bot-code]
+  (cheshire/generate-string {:code (:code bot-code)
+                             :state player-state}))
+
+(defn- lambda-invoke-request
```

Comment:
When I used threading, I kept getting a NullPointerException for this so I went back to what I knew was working.